src/process/process.cr

```diff
@@ -37,6 +38,19 @@ module Process
     LibC.getpid()
   end
 
+  def self.getpgid(pid : Int32)
+    gpid = LibC.getpgid(pid)
+    if gpid.is_a?(Int32) && gpid.to_i >= 0
```

Comment:
When using this inside a thread or a fork I sometime got an issue of

cannot do >= for Nil
so... I plan ahead :)